### PR TITLE
Snack+nav+typography

### PR DESCRIPTION
**Pull request by vp5h**
- SnackBar https://deploy-preview-5--avidcss.netlify.app/components/snackbar/snackbar
- Navigation https://deploy-preview-5--avidcss.netlify.app/components/navigation/navigation
- Typogarphy https://deploy-preview-5--avidcss.netlify.app/components/typography/typography
- Getting Started https://deploy-preview-5--avidcss.netlify.app/getting-started.html

### CODE REVIEW

File: Components/SnackBar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    padding: 0.5rem;
+    background: #333333;
+    color: white;
+    border-radius: 3px;
+    width: 24.18rem;
+    height: 2.81rem;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    visibility: hidden;
```
Comment (vermasavita) [inline Components/SnackBar/snackbar.css:11]: You can use cursor: pointer property here.

**Review by Aazam10 — APPROVED** [review]
Looks great

File: Components/SnackBar/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        </nav>
+    </header>
+    <section class="documentation">
+        <aside class="sidebar">
+            <div class="list-topic">
+                <h4 class="highlight">Get Started</h4>
+            </div>
+            <ul>
+                <li><a href="../../getting-started.html">Introduction</a></li>
+                <!-- <li><a href="../Colors/colors.html">Colors</a></li>
```
Comment (vermasavita) [inline Components/SnackBar/snackbar.html]: Remove these comments if not needed

File: Components/SnackBar/snackbar.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            </ul>
+        </aside>
+        <main class="main-content">
+            <div class="content-head">
+                <h2 class="highlight"> Snackbar</h2>
+            </div>
+            <div class="content">
+                <p>Snackbars provide brief messages about app processes at the bottom of the screen.</p>
+            </div>
+
```
Comment (vermasavita) [inline Components/SnackBar/snackbar.html:68]: Remove space

File: Components/SnackBar/snackbar.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+            <div class="divider"></div>
+            <div class="content-warp">
+
+                <div class="content-head">
+                    <h2 class="highlight">Baseline</h2>
+                </div>
+                <div class="content">
+
```
Comment (vermasavita) [inline Components/SnackBar/snackbar.html:77]: Remove space

File: Components/Navigation/navigation.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  background: var(--primary-color);
```
Comment (vermasavita) [inline Components/Navigation/navigation.css:5]: Use cursor here and on icons too.

File: Components/Typography/typography.html [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=vscode&wt=sharp&l=auto&width=680&ds=true&dsyoff=31px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522example%2522%253E%250A%2520%2520%253Cp%2520class%253D%2522txt-underline%2522%253EThis%2520is%2520text%2520underline%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522txt-strike%2522%253EThis%2520is%2520text%2520strikethrough%253C%252Fp%253E%250A%253C%252Fdiv%253E%250A"
+                        style=" height: 151px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                </div>
+            </div>
+
+
+
```
Comment (vermasavita) [inline Components/Typography/typography.html]: Remove these spaces

File: Components/Typography/typography.css [hunk 7]
```diff
@@ -0,0 +1,75 @@
+h1,
+.h1 {
```
Comment (vermasavita) [inline Components/Typography/typography.css:2]: Why did you use h1 & then .h1 ?

Is h1 a class?

**Review by vermasavita — APPROVED** [review]

File: Components/Typography/typography.css (hunk 7 above)
Comment (vp5h) [reply Components/Typography/typography.css:2]: yes

**Pull request merged**